cilium-enterprise-mixin/dashboards/cilium-operator.json

```diff
@@ -1084,19 +4264,20 @@
         "name": "prometheus_datasource",
         "options": [],
         "query": "prometheus",
+        "queryValue": "",
         "refresh": 1,
         "regex": "(?!grafanacloud-usage|grafanacloud-ml-metrics).+",
         "skipUrlSync": false,
         "type": "datasource"
       },
       {
         "current": {
-          "selected": false,
+          "selected": true,
```

Comment:
Same for this, please leave the defaults as `false`, `All`, `$__all`, etc to ensure it functions properly with new datasources and labels